dlog/commitment/src/commitment.rs

```diff
@@ -352,20 +353,30 @@ impl<G: CommitmentCurve> SRS<G> where G::ScalarField : CommitmentField {
             for (p_i, degree_bound) in plnms.iter().filter(|p| p.0.is_zero() == false) {
                 let mut offset = 0;
                 // iterating over chunks of the polynomial
-                while offset < p_i.coeffs.len() {
-                    let segment = DensePolynomial::<Fr<G>>::from_coefficients_slice
-                        (&p_i.coeffs[offset..if offset+self.g.len() > p_i.coeffs.len() {p_i.coeffs.len()} else {offset+self.g.len()}]);
-                    // always mixing in the unshifted segments
-                    p += &segment.scale(scale);
-                    scale *= &polyscale;
-                    offset += self.g.len();
-                    if offset > p_i.coeffs.len() {
-                        if let Some(m) = degree_bound {
+                if let Some(m) = degree_bound {
```

Comment:
What was the mistake here? Hard to parse out why this is changed in this way

Reply:
this change has been pr'd few weeks ago to go to the master. it is the same change.